voicevox_engine/synthesis_engine/make_synthesis_engine.py

```diff
@@ -47,7 +51,13 @@ def make_synthesis_engine(
             file=sys.stderr,
         )
 
-    core.initialize(voicelib_dir.as_posix() + "/", use_gpu)
+    if cpu_num_threads == 0:
+        print(
+            "Warning: cpu_num_threads is set to 0. "
+            + "( The library leaves the decision to the synthesis runtime )"
+        )
```

Comment:
他の所では標準エラー出力に出力しているので合わせた方がいいかもと思いました。

```suggestion
        print(
            "Warning: cpu_num_threads is set to 0. "
            + "( The library leaves the decision to the synthesis runtime )",
            file=sys.stderr,
        )
```

Reply:
ありがとうございます！修正しました